pyacq/core/tests/test_host.py

```diff
@@ -5,15 +5,15 @@
 from pyacq.core.processspawner import ProcessSpawner
 from pyacq.core.rpc import RPCClient
 
-logging.basicConfig(level=logging.INFO)
+logging.getLogger().level=logging.INFO
 
 
 def test_host1():
     
-    process_host0  = ProcessSpawner(Host,  'host0', 'tcp://127.0.0.1:5406')
-    process_host1  = ProcessSpawner(Host,  'host1', 'tcp://127.0.0.1:5407')
+    process_host0  = ProcessSpawner(Host,  'host0', 'tcp://127.0.0.1:*')
+    process_host1  = ProcessSpawner(Host,  'host1', 'tcp://127.0.0.1:*')
     
```

Comment:
In my mind, the Host port is fixed in advance, because a manager need to connect to Host.
It is more usefull for NodeGroup and in a case of only one Host on the same machine than the manager.